pdr-api/handlers/search/index.mjs

```diff
@@ -56,13 +56,29 @@ export const handler = async (event, context) => {
         term: {
           status: 'current'
         }
+      };
+    } else {
+      if (queryParams?.status) {
+        query.query.bool['filter'] = {
+          term: {
+            status: queryParams?.status
+          }
+        };
+      }
+      //TODO: Type to be implemented in #226, not currently in data model
+      if (queryParams?.type) {
+        query.query.bool['filter'] = {
+          term: {
+            type: queryParams?.type
+          }
+        };
```

Comment:
These need to be converted into a logical AND as there will be more than 1 filter/search criteria performed at a time.

Reply:
Addressed in [5663faa](https://github.com/bcgov/parks-data-register/pull/228/commits/5663faa7159301cd3174ab7a8fc036b134eebcc1)